UnitySDK/Assets/ML-Agents/Scripts/LearningBrain.cs

```diff
@@ -45,14 +44,17 @@ private void SetCommunicator(ICommunicator communicator)
             m_BatchedDecisionMaker = communicator;
             communicator?.SubscribeBrain(name, brainParameters);
             LazyInitialize();
-
         }
 
         /// <inheritdoc />
         protected override void Initialize()
         {
             var aca = FindObjectOfType<Academy>();
-            var comm = aca?.Communicator;
+            ICommunicator comm = null;
```

Comment:
Weird, it undid the `?.` here but added one below?

Reply:
Since Academy is a game object, it doesn't follow the regular Object lifecycle rules of a C# object.  So using the `?` on it doesn't really help determine whether or not it's a valid object to call.